zubhub_frontend/zubhub/src/assets/js/styles/views/login/loginStyles.js

```diff
@@ -1,127 +1,188 @@
-import { fade } from '@mui/material/styles';
-
-const styles = theme => ({
-  root: {
-    paddingTop: '2em',
-    paddingBottom: '2em',
-    flex: '1 0 auto',
-    // background: 'var(--primary-color2)',
-    // background:
-    //   'linear-gradient(to bottom, var(--primary-color2) 0%, var(--primary-color2) 25%, rgba(255,255,255,1) 61%, rgba(255,255,255,1) 100%)',
-  },
-  containerStyle: {
-    maxWidth: '600px',
-    [theme.breakpoints.up('1600')]: {
-      maxWidth: '950px',
+export const mainStyles = theme => ({
```

Comment:
why is this now `mainStyles`? we should revert back to `styles`. This only serves to add more diff and doesn't improve the code in any way